pset-iterators.js

```diff
@@ -160,7 +176,11 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
+const firstRepublican = presidents.filter(function (president) {
+  return president.party === 'Republican'
+})
 
+console.log(firstRepublican)
```

Comment:
@deedeemedina looks like you didn't get a chance to finish #7, take a look at the solution to get a sense of how I approached it
https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L201